plugin/plugin.go

```diff
@@ -37,7 +39,15 @@ func (p *plugin) Find(ctx context.Context, req *secret.Request) (*drone.Secret,
 	// to retrieve the secret at the requested path.
 	params, err := p.find(path)
 	if err != nil {
-		return nil, errors.New("secret not found")
+		if err, ok := err.(*api.ResponseError); ok {
+			var builder strings.Builder
+			for _, e := range err.Errors {
+				builder.WriteString("* " + e)
```

Comment:
Why this `*` foo? Just build a slice of error messages and join them by comma.

Reply:
I believe this was the same/similar format to what Vault itself used. I can change it to just comma separated instead